Do not attempt to download googletest; use the Guix-packaged one.

diff --git a/unittests/CMakeLists.txt b/unittests/CMakeLists.txt
index 505a000..e07c133 100644
--- a/unittests/CMakeLists.txt
+++ b/unittests/CMakeLists.txt
@@ -1,13 +1,3 @@
-include(FetchContent)
-FetchContent_Declare(
-  googletest
-  URL https://github.com/google/googletest/archive/refs/tags/v1.13.0.zip
-  URL_HASH SHA256=ffa17fbc5953900994e2deec164bb8949879ea09b411e07f215bfbb1f87f4632
-)
-# For Windows: Prevent overriding the parent project's compiler/linker settings
-set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
-FetchContent_MakeAvailable(googletest)
-
 include(GoogleTest)
 
 # RMA unit tests for instrumentation are only available if RMA and MPI are
diff --git a/unittests/rma/CMakeLists.txt b/unittests/rma/CMakeLists.txt
index fb16533..4e3da09 100644
--- a/unittests/rma/CMakeLists.txt
+++ b/unittests/rma/CMakeLists.txt
@@ -13,8 +13,9 @@ add_sources_to_format(SOURCES intersections.cpp)
 
 target_link_libraries(
   unit_tests_rma
-  GTest::gtest_main
-  GTest::gmock
+  gtest_main
+  gmock
+  gtest
   MPI::MPI_CXX
   RMADynamic
 )
